wordpress-mobile/WordPress-Android
# Add analytics to post list layout type toggle button

Fixes #9648

Adds tracking for toggle post list layout action in the toolbar on the Post List screen.

To test:
1. Go to Me -> App Settings -> Privacy Settings -> and make sure "Collect information" toggle is on
2. Go to My Site -> Blog Posts
3. Click on the "Toggle list layout type" button  in the Toolbar
4. Look in the LogCat or Tracks and make sure the correct event is being tracked
```
Tracked: post_list_view_layout_toggled, Properties: {"post_list_view_layout_type":"STANDARD"} Tracked: post_list_view_layout_toggled, Properties: {"post_list_view_layout_type":"COMPACT"}
```
Update release notes:

- The feature hasn't been released yet + there are no user facing changes
